styles.css

```diff
@@ -0,0 +1,3 @@
+* {
+    padding: 0;
+}
```

Comment:
- [ ]  As requested by the previous reviewer please style the h1 tag with color of your choice